src/Take.Blip.Builder.UnitTests/LazyInputTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Lime.Protocol;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Lime.Protocol.Serialization;
+using Shouldly;
+using Take.Blip.Builder.Models;
+using Takenet.Iris.Messaging.Resources.ArtificialIntelligence;
+using Xunit;
+using Input = Take.Blip.Builder.Models.Input;
+using Newtonsoft.Json;
+using Lime.Protocol.Serialization.Newtonsoft;
+
+#pragma warning disable 4014
+
+namespace Take.Blip.Builder.UnitTests
+{
+    public class LazyInputTests : FlowManagerTestsBase, IDisposable
+
+    {
+        [Fact]
+        public async Task ArtificialIntelligenceExtensionAnalyzeAsyncShouldBeExecuted()
+        {
+            // Arrange
+            var requestJson = "{\"dialogState\":\"STARTED\",\"intent\":{\"confirmationStatus\":\"NONE\",\"name\":\"PlanMyTrip\",\"slots\":{\"fromCity\":{\"confirmationStatus\":\"NONE\",\"name\":\"fromCity\"},\"SlotName\":{\"confirmationStatus\":\"NONE\",\"name\":\"string\",\"resolutions\":{\"resolutionsPerAuthority\":[{\"authority\":\"string\",\"status\":{\"code\":\"ER_SUCCESS_MATCH\"},\"values\":[{\"value\":{\"name\":\"string\",\"id\":\"string\"}}]}]},\"value\":\"string\"},\"toCity\":{\"confirmationStatus\":\"NONE\",\"name\":\"toCity\",\"value\":\"Chicago\"},\"travelDate\":{\"confirmationStatus\":\"NONE\",\"name\":\"travelDate\"}}},\"locale\":\"en-US\",\"reason\":\"USER_INITIATED\",\"requestId\":\"amzn1.echo-api.request.1\",\"timestamp\":\"2019-03-23T00:34:14.000Z\",\"type\":\"IntentRequest\"}";
+            var json = JsonConvert.DeserializeObject<Dictionary<string, object>>(requestJson);
+
+            var MessageWithAnalyzable = new Message()
```

Comment:
Change MessageWithAnalyzable var name to CamelCase because it is a local variable and not a property

Reply:
Using camelCase fot local variables